### PR TITLE
allow specifying a default request id prefix, so all requests made from the same "project" can be tracked easily

### PR DESCRIPTION
**Pull request by tommyjcarpenter**

### CODE REVIEW

File: merqube_client_lib/constants.py [hunk 1]
```diff
@@ -12,3 +12,5 @@
 MERQ_RUN_ID_ENV_VAR: Final[str] = "MERQ-RUN-ID"
 
 DEFAULT_CACHE_TTL: Final[int] = 600
+
+MERQ_CLIENT_PREFIX = "merqube_py_cli"
```
Comment (aschmied) [inline merqube_client_lib/constants.py:16]: maybe merqube_py_client? cli usually means "command line interface"

File: tests/unit/test_indexapi_client_single.py [hunk 2]
```diff
@@ -192,3 +196,57 @@ def test_update(monkeypatch):
         call("/index/testid", json={"description": "test"}),
         call("/index/testid", json={"description": "test", "status": exis_status}),
     ]
+
+
+@freeze_time(now := pd.Timestamp.now())
+@pytest.mark.parametrize("already_locked", [True, False])
+def test_lock(monkeypatch, already_locked):
```
Comment (aschmied) [inline tests/unit/test_indexapi_client_single.py:203]: Did you intend to commit these? (not related to the current PR, so just making sure)

**Review by aschmied — DISMISSED** [review]

File: tests/unit/test_indexapi_client_single.py (hunk 2 above)
Comment (tommyjcarpenter) [reply tests/unit/test_indexapi_client_single.py:203]: yes, unrelated tests agree

File: merqube_client_lib/constants.py (hunk 1 above)
Comment (tommyjcarpenter) [reply merqube_client_lib/constants.py]: changed

**Pull request merged**